tailwind.config.cjs

```diff
@@ -12,7 +15,10 @@ module.exports = {
         primary: primaryColor,
         secondary: secondaryColor,
         tertiary: tertiaryColor,
-        tertiary60: tertiary60Color
+        tertiary60: tertiary60Color,
+        quaternary: quaternaryColor,
+        quaternary54: quaternary54Color,
+        lightRed: lightRedColor
```

Comment:
Maybe look into [Background Opacity](https://v2.tailwindcss.com/docs/background-opacity) but it only works for background or even [Customizing colors](https://tailwindcss.com/docs/customizing-colors#using-css-variables) that suports an alpha placeholder in order to remove some repeated colors